packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -676,6 +676,36 @@ class WebViewController {
   Future<String> getTitle() {
     return _webViewPlatformController.getTitle();
   }
+
+  /// Set the scrolled position of this view.
+  ///
+  /// 'x' - the x position, in pixels, to scroll to.
+  /// 'y' - the y position, in pixels, to scroll to.
+  Future<void> scrollTo(int x, int y) {
+    return _webViewPlatformController.scrollTo(x, y);
+  }
+
+  /// Move the scrolled position of this view.
+  ///
+  /// 'x' - the amount of pixels to scroll by horizontally.
+  /// 'y' - the amount of pixels to scroll by vertically.
+  Future<void> scrollBy(int x, int y) {
+    return _webViewPlatformController.scrollBy(x, y);
+  }
+
+  /// Return the horizontal scroll position of this view.
+  ///
+  /// Scroll position is measured from left.
```

Comment:
Mention the returned unit here and on `getScrollY`

Reply:
, in WebView pixels,